scripts/check-tags.py

```diff
@@ -0,0 +1,131 @@
+import re
+import sys
+
+# 注意：本脚本只支持跳过检查 ``` ``` 格式的代码块，~~~ ~~~ 格式的代码块会被检查。
```

Comment:
How about using code comments in English for this **docs** repo, and keep the Chinese comments for the **docs-cn** repo? So as to make the language friendly in case someone reads this file.

Reply:
The code comments are basically for me only. 😂 I‘ve deleted the Chinese comments here and will keep them in [docs-cn](https://github.com/pingcap/docs-cn/pull/4417).